### DISCUSSION TITLE
The android app run for a while, and it will be killed，how to solve this problem?

### DISCUSSION THREAD

**Discussion by ccinsz** [beeware/briefcase#1422] [Q&A]
The android app run for a while, and it will be killed，how to solve this problem?

**Comment by freakboy3742** [comment]
When you go to see a doctor, do you say "Doctor, I hurt. What is wrong"?

No - you say "Doctor, I was playing tennis yesterday, and after playing for about half an hour, my elbow started to hurt every time I did a backhand". 

The first response gives the doctor nothing to work with. The latter tells them what you were doing, how long you were doing it for, and the *specific* problem you're experiencing. Given that information, they can start to diagnose your problem.

The same is true of programming and debugging.

Without knowing what you are doing, and exactly what behavior you're seeing, it's impossible to diagnose and provide suggestions. If you think you've found a bug, that generally means producing a reproduction case - the simplest possible version of an app that will manifest the problem you're describing.

To be clear - that does *not* mean "send us the full source code for your entire app". It means build up a helloworld example until it has the problem you're seeing. We're looking for the simplest possible version of an application that exhibits the problem you're describing.

It also means providing *log files* - evidence that you're actually seeing the problem that you describe.

**Reply by ccinsz** [reply]
Sorry for not being clear.  Put the app in the background, but it still run the tasks. I find the log of the phone which writes "Killing 31136:com.example.demo/u0a228 (adj 700): excessive cpu 19520 during 300020 dur=1074673 limit=2".

**Reply by freakboy3742** [reply]
Again - *the* app? *What* app? Any app? Your app? 

If what you're reporting is that the app doesn't continue running in the background - that's expected behavior. Phones don't run like desktop machines - there's no guarantee that an app keeps running while it's not in the foreground.

If that's not what you're reporting, then I once again need to ask you for *specific* details and a *specific* reproduction case, with *full* logs for what is going on.

**Reply by ccinsz** [reply]
Got it, and use other way to solve it. Thank you for your help.
